spec/itunes/volume_spec.rb

```diff
@@ -0,0 +1,47 @@
+# coding: utf-8
+require 'spec_helper'
+require 'itunes-client'
+
+include Itunes
+
+describe Volume do
+  let(:volume) { Itunes::Player.volume }
+  let(:level) { Fixnum }
+
+  describe '.up' do
+    it 'calls up.scpt' do
+      expect(Volume).to receive(:execute_script).with('volume/up.scpt', level)
+      Volume.up
+    end
+  end
+
+  describe '.down' do
+    it 'calls down.scpt' do
+      expect(Volume).to receive(:execute_script).with('volume/down.scpt', level)
+      Volume.down
+    end
+  end
+
+  describe '.mute' do
+    it 'calls mute.scpt' do
+      expect(Volume).to receive(:execute_script).with('volume/mute.scpt')
+      Volume.mute
+    end
+  end
+
+  describe '.unmute' do
+    it 'calls unmute.scpt' do
+      expect(Volume).to receive(:execute_script).with('volume/unmute.scpt')
+      Volume.unmute
+    end
+  end
+
+  describe '.value' do
+    subject { volume.value }
```

Comment:
You can remove context block because you don't need more than two contexts.
And it looks this will pass CI.

```ruby
  describe '.value' do
    let(:value) { 30 }
    before { expect(Volume).to receive(:execute_script).with('volume/value.scpt').and_return(value) }
    it { expect(Volume.value).to eq(value) }
  end
```

Your code `and_return(volume.value)` calls real method so CI fails.